test/test_jit.py

```diff
@@ -269,6 +269,28 @@ def _sum_of_list(tensorlist):
         s += t.sum()
     return s
 
+# helper function to generate test qparam
+
+
+def _helper_generate_qparam(trace, x1):
```

Comment:
Minor nit: Can we call the input as 'input_data' not x1?. Also, can we call the input model, my_script_model instead of trace as we arent tracing it?